.github/workflows/ci.yml

```diff
@@ -39,3 +39,23 @@ jobs:
 #        uses: github/super-linter@v4
 #        env:
 #          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  test:
```

Comment:
Not a fan of naming here.

IMO this should be a separate CI workflow and CI job should be named like `Integration tests` or smth as these are not unit tests. Also, might need to mention that these desktop Chrome tests as in future there will be different kind of tests

Reply:
Yes, I agree, I will move it then